### PR TITLE
Add reverse reverse colourmap for sliceviewer

### PR DESCRIPTION
**Pull request by Harrietbrown**
**Description of work.**

This PR adds a checkbox to sliceviewer that reverses the colourmap. because of this it also removes the _r reverse maps form the combo box.

**To test:**

1. In workbench load some data and show slice viewer
2. check the dropdown menu, there should be no options ending in `_r`
3. check that changing the option changes the colourmap
4. change that ticking the reverse box reverses the coloumap
5. in setting->plots change the default colourmap and check that sliceviewer uses that default.

re: #28915 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v5.1.0/mantidworkbench.rst
Comment (ConorMFinn) [inline docs/source/release/v5.1.0/mantidworkbench.rst:87]: Was this added in a previous PR but not added to the release notes at the time?

**Review by cailafinn — COMMENTED** [review]
Code looks good and works well, just one thing I want to check before approval

File: docs/source/release/v5.1.0/mantidworkbench.rst
Comment (Harrietbrown) [reply docs/source/release/v5.1.0/mantidworkbench.rst:87]: The PR that put in the default colourbar says it for image plots (line 84) I wanted to clarify it was for slice viewer as well.

**Review by cailafinn — APPROVED** [review]
Code looks good and changes work as described. Good work 👍

**Pull request merged**
